### DISCUSSION TITLE
I'm getting a computer to run pgsc_calc. What specs do you suggest?

### DISCUSSION THREAD

**Discussion by Fiwx** [PGScatalog/pgsc_calc#229] [Q&A] [answered]
Hello,
I'm purchasing a computer to run pgsc_calc (and imputation). The minimum requirements are:

- 16GB RAM
- 2 CPUs

However, I was wondering what "decent" instead of minimum specs would be, in terms of RAM, CPU amount/speed, or anything else you think is important. Thank you!

**Comment by nebfield** [comment, marked as answer]
It depends on the size of your input data 😄 

I personally use an M1 Macbook with 32GB of RAM to develop the workflow and never have any problems. I test with datasets like 1000 Genomes and the Human Genome Diversity Project. 

However for really big jobs - like calculating scores for 500,000 people in UK Biobank - it's best to get access to a HPC cluster, cloud account, or a desktop with a bunch of RAM (128GB +). 

Lots of fast storage is also very helpful. In order fastest to slowest: NVMe > SSD > HDD

**Reply by Fiwx** [reply]
Thanks! My input data is about 1000 genomes. It would be great if it could finish in a few days. I will look for desktops with NVMe and decent RAM. I think one of these two might work:
https://www.amazon.com/dp/B0C6Y8XKK4?th=1
https://www.amazon.com/dp/B0CK4BLSJD?th=1

Do you think I'm missing anything? I appreciate the tips!
